networkgen/templates/networkgen/generator_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+{% block content %}
+<h2>Upload a .sdf file</h2>
+<form method="post" enctype="multipart/form-data">
+{% csrf_token %}
+<input type="file", name="in_sdf">
```

Comment:
See the comment in networkgen/forms.py. Half this code is unneeded here if the Form is correctly build there.